lms/djangoapps/teams/tests/test_views.py

```diff
@@ -456,6 +459,38 @@ def test_expand_public_user(self):
         )
         self.verify_expanded_public_user(result['results'][0]['membership'][0]['user'])
 
+    def test_text_search_name(self):
```

Comment:
How about a test for overlapping terms, and also one for when search should return nothing?